libraries/hotstuff/test/test_hotstuff.cpp

```diff
@@ -173,6 +208,45 @@ BOOST_AUTO_TEST_CASE(hotstuff_bitset) try {
 } FC_LOG_AND_RETHROW();
 
 
+static std::vector<fc::crypto::blslib::bls_private_key> map_to_sks(std::vector<std::string> keys){ 
+
+   std::vector<fc::crypto::blslib::bls_private_key> sks;
+
+   std::transform(keys.cbegin(), keys.cend(), std::back_inserter(sks),
+                   [](std::string k) { return fc::crypto::blslib::bls_private_key(k); });
+   
+   return sks;
+  
+}
+
+static finalizer_set create_fs(std::vector<std::string> keys){
+   
+   std::vector<fc::crypto::blslib::bls_private_key> sks;
+
+   std::vector<finalizer_authority> f_auths;
+
+   f_auths.reserve(keys.size());
+
+   for (const auto& urk : keys){
+
+      fc::crypto::blslib::bls_private_key sk = fc::crypto::blslib::bls_private_key(urk);
+      fc::crypto::blslib::bls_public_key pk = sk.get_public_key();
+
+      sks.push_back(sk);
+
+      f_auths.push_back(eosio::chain::finalizer_authority{"" , 1 , pk});
+
+   }
+  
+   eosio::chain::finalizer_set fset;
+
+   fset.fthreshold = 15;
+   fset.finalizers = f_auths;
+
+   return fset;
+
+}
+
```

Comment:
Code style... we usually don't have all these emply lines in our code, I think most of these could be removed.